README.md

```diff
@@ -5,75 +5,16 @@
 ![Lint](https://github.com/aquarist-labs/s3gw/actions/workflows/lint.yaml/badge.svg)
 ![Build](https://github.com/aquarist-labs/s3gw/actions/workflows/build-environment.yml/badge.svg)
 
-An S3-compatible Gateway based on Ceph RGW, using a non-RADOS backend for
-standalone usage.
-
-This project provides the required infrastructure to build a container
-able to run on a kubernetes cluster, providing S3-compatible endpoints to
-applications.
-
-## Table of Contents
-
-- [s3gw](#s3gw)
-- [Roadmap](#roadmap)
-- [Quickstart](#quickstart)
-  - [Helm chart](#helm-chart)
-  - [Podman](#podman)
-  - [Docker](#docker)
-- [License](#license)
-- [Developing the s3gw](#developing-the-s3gw)
-
-## Roadmap
-
-![Roadmap](/assets/images/s3gw-roadmap.jpg)
```

Comment:
Are we removing the roadmap here on purpose? Where will it be living otherwise?

Reply:
yes, it'll live in the documentation here: https://s3gw-docs.readthedocs.io/en/latest/ - once this PR is merged, I'll be able to generate the new documentation